internal/controller/config/config.go

```diff
@@ -114,7 +115,7 @@ func (cm *NumaflowControllerDefinitionsManager) UpdateNumaflowControllerDefiniti
 	for _, controller := range config.ControllerDefinitions {
 		cm.rolloutConfig[controller.Version] = controller.FullSpec
 
-		fmt.Printf("Added/Updated Controller definition Config, version: %s\n", controller.Version)
+		log.Debug().Msg(fmt.Sprintf("Added/Updated Controller definition Config, version: %s", config)) // due to cyclical dependency, we can't call logger
```

Comment:
good solution to the problem of the cyclical dependency :) I forgot about these print statements